### DISCUSSION TITLE
Support with 2FA for kosher phones owners

### DISCUSSION THREAD

**Discussion by danielap1996** [community/community#128932] [Mobile]
### Support for Kosher phone owners
HI
for some project in which I should work on my work we can get access only with 2FA validation
I , and more workers, have a kosher phone (https://www.usmobile.com/blog/kosher-phone-plans-understanding-the-basics/)
which mean that we don't have apps, and we don't get SMS
we need to get a support to connect the project with the 2FA validation
other servers , in most of the time, offer a phone call in which we can here the code instead of getting SMS

**Comment by mattruggio** [comment]
Hey @danielap1996 - I might need a bit of clarification here: Are you specifically trying to log into the mobile app on an Android or iOS device but are stuck during the 2FA process?

**Comment by DAVIDKIMGANGMIN2816** [comment]
PRINT C
